website/docs/r/kusto_table.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+subcategory: "Data Explorer"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_kusto_table"
+description: |-
+  Manages a Data Explorer.
+---
+
+# azurerm_kusto_table
+
+Manages a Data Explorer.
+
+## Example Usage
+
+```hcl
+resource "azurerm_kusto_table" "example" {
+  name = "example"
+  database_id = "TODO"
+
+  column {
+    name = "example"
+    type = "TODO"    
+  }
+}
```

Comment:
in general, we will list all the dependency in the example usage and remember to format it. you could use the tool: terrafmt

Reply:
fixed